nautobot/extras/models/models.py

```diff
@@ -245,6 +248,7 @@ class ConfigContextSchema(OrganizationalModel):
         null=True,
         blank=True,
     )
+    # todoindex:
```

Comment:
```
nautobot/extras/datasources/git.py-837-        schema_record = ConfigContextSchema.objects.get(
nautobot/extras/datasources/git.py-838-            name=schema_metadata["name"],
nautobot/extras/datasources/git.py-839-            owner_content_type=git_repository_content_type,
nautobot/extras/datasources/git.py:840:            owner_object_id=repository_record.pk,
nautobot/extras/datasources/git.py-841-        )
nautobot/extras/datasources/git.py-842-    except ConfigContextSchema.DoesNotExist:
nautobot/extras/datasources/git.py-843-        schema_record = ConfigContextSchema(
nautobot/extras/datasources/git.py-844-            name=schema_metadata["name"],
nautobot/extras/datasources/git.py-845-            slug=slugify(schema_metadata["name"]),
nautobot/extras/datasources/git.py-846-            owner_content_type=git_repository_content_type,
nautobot/extras/datasources/git.py:847:            owner_object_id=repository_record.pk,
--
nautobot/extras/datasources/git.py-893-    for schema_record in ConfigContextSchema.objects.filter(
nautobot/extras/datasources/git.py-894-        owner_content_type=git_repository_content_type,
nautobot/extras/datasources/git.py:895:        owner_object_id=repository_record.pk,
nautobot/extras/datasources/git.py-896-    ):
```

Reply:
For modifying the compound indexing on this model, please be sure to leverage the `Meta.constraints` form vs. the legacy `Meta.unique_together` or `Meta.indexes vs. `Meta.index_together`:

See: https://docs.djangoproject.com/en/4.0/ref/models/options/#django.db.models.Options.constraints

In this case you'd want `Meta.contstraints` with a `UniqueConstraint` defined.

Reply:
To add an index to `owner_object_id`, this is going to be done by the way of using the `Meta.constraints` of `UniqueConstraint` across `name`, `owner_object_id`, `owner_content_type`.

Additionally, please remove the `unique=True` on `name`.